egs/fisher_english/s5/local/semisup/chain/tuning/run_tdnn_15k_semisupervised_conf_am.sh

```diff
@@ -0,0 +1,418 @@
+# Deriv weights: Lattice posterior of best path pdf
+# Unsupervised weight: 1.0
+# Weights for phone LM (supervised, unsupervised): 5,2
+# LM for decoding unsupervised data: 4gram
+# Supervision: Naive split lattices
+
+set -u -e -o pipefail
+
+stage=0   # Start from -1 for supervised seed system training
+train_stage=-100
+nj=40
+decode_nj=40
+exp=exp/semisup_15k
+
+# Datasets -- Expects data/$supervised_set and data/$unsupervised_set to be 
+# present
+unsupervised_set=train_unsup100k_250k  # set this to your choice of unsupervised data
+supervised_set=train_sup15k
+semisup_train_set=semisup15k_250k
+
+# Seed model options
+nnet3_affix=_semi15k_250k    # affix for nnet3 and chain dir -- relates to i-vector used
+tdnn_affix=7j  # affix for the supervised chain-model directory
+train_supervised_opts="--stage -10 --train-stage -10"
+
+# Unsupervised options
+decode_affix=
+egs_affix=  # affix for the egs that are generated from unsupervised data and for the comined egs dir
+unsup_frames_per_eg=150  # if empty will be equal to the supervised model's config -- you will need to change minibatch_size for comb training accordingly
+lattice_lm_scale=0.5  # lm-scale for using the weights from unsupervised lattices
+lattice_prune_beam=4.0  # If supplied will prune the lattices prior to getting egs for unsupervised data
+tolerance=1
+phone_insertion_penalty=
+
+rescore_unsup_lattices=true
+unsup_rescoring_affix=big
+
+# Semi-supervised options
+comb_affix=comb1am  # affix for new chain-model directory trained on the combined supervised+unsupervised subsets
+supervision_weights=1.0,1.0
+lm_weights=5,2
+
+sup_egs_dir=   # Supply this to skip supervised egs creation
+unsup_egs_dir=  # Supply this to skip unsupervised egs creation
+unsup_egs_opts=  # Extra options to pass to unsupervised egs creation
+
+# Neural network opts
+apply_deriv_weights=true
+xent_regularize=0.1
+hidden_dim=725
+minibatch_size="150=128/300=64"
+
+decode_iter=
+
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+egs_affix=${egs_affix}_prun${lattice_prune_beam}_lmwt${lattice_lm_scale}_tol${tolerance}
+
+RANDOM=0
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+if [ $stage -le -1 ]; then
+  echo "$0: chain training on the supervised subset data/${supervised_set}"
+  local/chain/run_tdnn_15k_j.sh $train_supervised_opts \
+                          --train-set $supervised_set \
+                          --unsup-train-set $unsupervised_set \
+                          --semisup-train-set $semisup_train_set \
+                          --nnet3-affix "$nnet3_affix" --tdnn-affix "$tdnn_affix" --exp $exp || exit 1
+fi
+
+lang=data/lang_chain
+unsup_decode_lang=data/lang_poco_test_ex250k
```

Comment:
add brief description for rescore LM and decode LM! Did you generate poco LM using 15 hrs training data?